openshift2nulecule/openshift.py

```diff
@@ -124,7 +124,15 @@ def export_project(self):
             ec, stdout, stderr = self._call_oc(args)
             objects = anymarkup.parse(stdout, format="json", force_types=None)
 
-            ep = ExportedProject(artifacts=objects)
+            # convert OpenShift List to array
+            if objects["kind"] == "List":
+                artifacts = objects["items"]
+            else:
+                msg = "Output of oc export command is diferent Kind than List"
```

Comment:
Can this line be:
`"Output of 'oc export' command is different kind than 'List'"`